test.properties

```diff
@@ -7669,22 +7686,20 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
         modules-integration-bundle-oracle193-jdk21_zulu,\
         modules-integration-bundle-postgresql121-jdk21_zulu,\
         modules-integration-bundle-sqlserver2017-jdk21_zulu,\
-        portal-renamed-tomcat-in-path-mysql57-jdk8,\
-        portal-startup-space-in-path-mysql57-jdk8
+        portal-renamed-tomcat-in-path-mysql57-jdk21_zulu,\
+        portal-startup-space-in-path-mysql57-jdk21_zulu
 
     test.batch.names[portal-release][portal]=\
-        empty-osgi-core-dir-mysql57-jdk8,\
-        functional-bundle-tomcat-hypersonic20-jdk8,\
-        functional-bundle-tomcat-mariadb102-jdk8,\
-        functional-bundle-tomcat-mysql57-jdk11_zulu,\
-        functional-bundle-tomcat-mysql57-jdk17_open,\
+        empty-osgi-core-dir-mysql57-jdk21_zulu,\
+        functional-bundle-tomcat-hypersonic20-jdk21_zulu,\
+        functional-bundle-tomcat-mariadb102-jdk21_zulu,\
         functional-bundle-tomcat-mysql57-jdk21_zulu,\
-        functional-bundle-tomcat-mysql80-jdk8,\
-        functional-bundle-tomcat-postgresql121-jdk8,\
+        functional-bundle-tomcat-mysql80-jdk21_zulu,\
+        functional-bundle-tomcat-postgresql121-jdk21_zulu,\
         functional-bundle-ubuntu18-tomcat90-mysql57-jdk8,\
-        functional-smoke-bundle-wildfly180-mysql57-jdk8,\
-        functional-smoke-bundle-wildfly230-mysql57-jdk8,\
-        functional-smoke-bundle-wildfly261-mysql57-jdk8,\
+        functional-smoke-bundle-wildfly180-mysql57-jdk21_zulu,\
+        functional-smoke-bundle-wildfly230-mysql57-jdk21_zulu,\
+        functional-smoke-bundle-wildfly261-mysql57-jdk21_zulu,\
         patching-tool-mysql57-jdk8,\
         playwright-js-bundle-tomcat90-mysql57-jdk8
```

Comment:
@vicnate5 do you mean to change this to `playwright-js-tomcat90-mysql57-jdk21_zulu` /.

Reply:
@julschong Yes